### PR TITLE
Added manifest file for plugin cluster-compare

### PR DESCRIPTION
**Pull request by yangzhengtx**
This is the pull request for merging plugin of cluster-compare.

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

**Comment by ahmetb** [comment]
Hi, thanks for the submission. Are you one of the maintainers of the project? Or are you distributing this tool on their behalf?

**Comment by yangzhengtx** [comment]
Hi, I am creating this PR on the team's behalf. We have one team to develop this tool.

**Comment by ahmetb** [comment]
Understood. Are the maintainers aware that this plugin is distributed on Krew?

Also, you need to sign the CLA to contribute to Kubernetes orgs.

**Comment by yangzhengtx** [comment]
Thank you. @ahmetb I have signed the CLA.
The maintainers are aware that the plugin is distributed on Krew.

File: plugins/cluster-compare.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cluster-compare
+spec:
+  version: v0.0.1
+  homepage: https://github.com/openshift/kube-compare
+  shortDescription: For checking that a configuration is as-expected.
```
[the remark this answers is not in the document]
Comment (yangzhengtx) [reply plugins/cluster-compare.yaml:8]: Sure. I can change the description to that.

File: plugins/cluster-compare.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/openshift/kube-compare/releases/download/v0.0.1/kube-compare_linux_amd64.tar.gz
+    sha256: 174c6dfc407718deaa9f1316cf2830099dbffbbbbe3dd0ec5fcb141c2972bc72
+    bin: kubectl-cluster_compare
+  - selector:
+      matchLabels:
+        os: linux
+        arch: 386
```
Comment (ahmetb) [inline plugins/cluster-compare.yaml:32]: Instead of 386 I'd recommend adding darwin/arm64, windows/arm64, linux/arm64. I don't know who still has 386. :)

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks! I left an extra comment to fix in a subsequent version. 


Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

File: plugins/cluster-compare.yaml (hunk 2 above)
Comment (yangzhengtx) [reply plugins/cluster-compare.yaml:32]: Thank you, I have the same concerns. I will discuss this with our team. :)

**Pull request merged**